fields-livedata/src/commonMain/kotlin/dev/icerock/moko/fields/FormField.kt

```diff
@@ -30,17 +33,21 @@ open class FormField<D, E>(initialValue: D, validation: (LiveData<D>) -> LiveDat
             if (show) error else null
         }
     }
+
     val isValid: LiveData<Boolean> by lazy {
         validationError.map { it == null }
     }
 
-    fun setError(error: E?) {
+    override val isValidValue: Boolean
+        get() = isValid.value
```

Comment:
why we should add it?

Reply:
made a Boolean return value for the validate() function

Reply:
cool